docs/glossary.md

```diff
@@ -0,0 +1,101 @@
+# we can just set it to an arbitrarily high value.
+sidebar_position: 999
+---
+
+# Glossary
+
+This glossary provides definitions of terms specific to Divvi Up and the
+protocols it uses.
+
+### Aggregator
+
+A server that implements the aggregation subprotocol of
+[DAP](#distributed-aggregation-protocol-dap). Divvi Up and its partners operate
+DAP-compliant aggregators. Aggregators can also be self-hosted.
+
+Aggregators ingest measurements sent from [clients](#client). They aggregate
+reports into batches, and service [collector](#collector) requests.
+
+Aggregators can be in one of two roles: [Leader](#leader) or [Helper](#helper).
+
+Examples of aggregator software include [Janus (ISRG)][janus], and [Daphne
+(Cloudflare, helper-only)][daphne].
+
+### Client
+
+An end-user app that uses the client subprotocol of
+[DAP](#distributed-aggregation-protocol-dap) to contribute measurements to a
+[task](#task). Typically, app developers will integrate a DAP client library to
+achieve this.
+
+### Collector
+
+An application that implements the collector subprotocol of
+[DAP](#distributed-aggregation-protocol-dap). It collects aggregated batches
+from the aggregators and provides the final aggregate to the
+[subscriber](#subscriber). This is most often operated by the application
+developer.
+
+A library suitable for building collectors is found in the Rust crate
+[`janus_collector`][janus_collector]. A primitive collector implementation is
+available in [Janus][collect.rs].
+
+### Distributed Aggregation Protocol (DAP)
+
+A protocol that describes a multi-party distributed system for privacy
+preserving aggregation of measurements. It is currently in the draft phase. Its
+specification can be viewed on the [IETF data tracker][DAP].
+
+Each participant in a DAP system adopts one of these roles: [client](#client),
+[collector](#collector), [leader](#leader), or [helper](#helper).
+
+### Helper
+
+A [DAP](#distributed-aggregation-protocol-dap) aggregator that performs
+aggregation and collection in response to direction from the Leader.
+
+The system requirements for a helper are lower than those for a leader.
+
+### Leader
+
+A [DAP](#distributed-aggregation-protocol-dap) aggregator that directly ingests
+[reports](#report) from clients, and drives the aggregation and collection
+processes with the helper. The leader determines how reports are to be batched.
+
+The system and network bandwidth requirements for a leader are higher than for
+those for a helper.
+
+### Query type
+
+Describes how [reports](#report) are grouped into batches for aggregation. There
+are two query types defined by [DAP](#distributed-aggregation-protocol-dap):
+
+- Time-interval: Reports are grouped into batches based on the timestamp of the
+  report.
+- Fixed-size: Reports are grouped into arbitrary batches by the leader.
+
+### Report
+
+A client-submitted measurement. Each report is composed of two encrypted report
+shares: one for the leader, and one for the helper.
```

Comment:
More or less yes, as IIUC proof shares are specific to Prio3?

Perhaps more detail is justified here, especially since this is missing discussion of the public share. Let me take a stab at it.

Reply:
I added detail about the composition of the report, according to the struct definition https://www.ietf.org/archive/id/draft-ietf-ppm-dap-04.html#name-leader-initialization.

I incorporated discussion of proofs in step 4 of How It Works.